src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+
+  switch (true) {
+    case days >= 7
+      : cost = (cost * days) - 50;
+      break;
+    case days >= 3 && days < 7
```

Comment:
```suggestion
    case days >= 3
```
Другу перевірку можна прибрато. ЯКщо днів буде > 7, то спрацює перший кейс і до цого черга не дійде